cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,30 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import ProfilePageObject from '../support/pages/profile.pageObject';
+
+const profilePage = new ProfilePageObject();
+
 describe('User', () => {
-  before(() => {
+  let user1;
+  let user2;
 
-  });
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user1 = generateUser;
+      cy.register(user1.email, user1.username, user1.password);
+    });
 
-  it.skip('should be able to follow the another user', () => {
+    cy.task('generateUser').then((generateUser) => {
+      user2 = generateUser;
+      cy.register(user2.email, user2.username, user2.password);
+    });
```

Comment:
Consider combining the user generation and registration steps for `user1` and `user2` into a single `cy.task` call to improve test efficiency. This can help reduce redundant code and speed up the test setup.